lib/Marlin/Marlin/src/module/endstops.cpp

```diff
@@ -87,6 +87,9 @@ Endstops::esbits_t Endstops::live_state = 0;
 #if ENABLED(SPI_ENDSTOPS)
   Endstops::tmc_spi_homing_t Endstops::tmc_spi_homing; // = 0
 #endif
+#if ENABLED(IMPROVE_HOMING_RELIABILITY)
+  millis_t sg_guard_period; // = 0
+#endif
```

Comment:
I intentionally removed it in https://github.com/prusa3d/Prusa-Firmware-Buddy/pull/2276/files

Reply:
I introduced an extra define HOMING_SG_GUARD_DURATION (which we don't use) to elide the code instead.